### ISSUE TITLE
No logs of the app failing to start

### ISSUE THREAD

**Issue by alexander-potemkin** [caprover/caprover#1140]
What is the problem?
CapRover doesn’t show the logs when Docker is failing to start, because the main app is failing. I can only get it with `docker ps` && `docker logs id`, while it would be nice to have this or some error message, showing the container current status and if it’s restarted like 5 times in the last 10 seconds.

Steps to reproduce the problem:
Create the Docker with failing process. Or just take any existing app and let it fail at the fail at start, with an exception.

**Comment by githubsaturn** [comment]
Does `docker service logs yourservice` output the logs?

**Comment by alexander-potemkin** [comment]
Yes, it does - that’s how I found out what went wrong (did ssh, docker ps, docker logs).

**Comment by louis77** [comment]
Had the same issue here. App was provided with some wrong args in the Dockerfile and didn't start. I only found out after a `docker service ls` command that it wasn't running. There was no log output in the UI.
